tests/metabase/management/__snapshots__/test_populate_metabase_matomo.ambr

```diff
@@ -1443,14 +1532,18 @@
     "\t> fetching date=2022-06-13 dashboard='tb 216 - Les femmes dans l'IAE' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/femmes-iae/",
     "\t> fetching date=2022-06-13 dashboard='tb 217 - Suivi pass IAE' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/suivi-pass-iae/",
     "\t> fetching date=2022-06-13 dashboard='tb 218 - Cartographie de l'IAE' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/cartographies-iae/",
+    "\t> fetching date=2022-06-13 dashboard='tb 287 - Conventionnements IAE' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/conventionnements-iae/",
     "\t> fetching date=2022-06-13 dashboard='tb 306 - Zoom sur les ESAT' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/zoom-esat/",
     "\t> fetching date=2022-06-13 dashboard='tb 32 - Acceptés en auto-prescription' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/auto-prescription/",
     "\t> fetching date=2022-06-13 dashboard='tb 336 - Suivi des prolongations' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/suivi-demandes-prolongation/",
     "\t> fetching date=2022-06-13 dashboard='tb 43 - Statistiques des emplois' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/statistiques-emplois/",
     "\t> fetching date=2022-06-13 dashboard='tb 52 - Typologie de prescripteurs' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/zoom-prescripteurs/",
     "\t> fetching date=2022-06-13 dashboard='tb 54 - Typologie des employeurs' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/zoom-employeurs/",
     "\t> fetching date=2022-06-13 dashboard='tb 90 - Analyse des métiers' pageUrl=https://pilotage.inclusion.beta.gouv.fr/tableaux-de-bord/metiers/",
-    '> about to fetch count=16 public dashboards from Matomo.',
+    '> about to fetch count=17 public dashboards from Matomo.',
+    'For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500',
```

Comment:
Pourquoi ici tu as ajouté 3 fois `'For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500',`

Reply:
C'est pas moi, en tout cas pas manuellement ;). Les fichiers dans les dossiers `__snapshots__` sont utilisé et géré par [syrupy](https://tophat.github.io/syrupy/).
Le test associé au snapshot vérifie la sortie de la commande `populate_metabase_matomo.py` pour s'assurer qu'elle fait bien ce qu'on veux, et la duplication des lignes est dû au fait qu'on essaie 3 fois (3 erreurs) l'appel à l'API Matomo : 
https://github.com/gip-inclusion/les-emplois/blob/6a1c9b042719530d55921799fda4a5671511e238/itou/metabase/management/commands/populate_metabase_matomo.py#L85